### PR TITLE
add compat data for SVG <hkern>

### PR DESCRIPTION
**Pull request by solfen**
Link: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/hkern

### CODE REVIEW

File: svg/elements/hkern.json [hunk 1]
```diff
@@ -0,0 +1,289 @@
+{
+  "svg": {
+    "elements": {
+      "hkern": {
```
Comment (teoli2003) [inline svg/elements/hkern.json]: Please put the attributes in lexicographic order.

File: svg/elements/hkern.json [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+                "version_added": null
+              },
+              "safari_ios": {
+                "version_added": null
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
```
Comment (teoli2003) [inline svg/elements/hkern.json]: true

File: svg/elements/hkern.json [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
```
Comment (teoli2003) [inline svg/elements/hkern.json]: true

File: svg/elements/hkern.json [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
```
Comment (teoli2003) [inline svg/elements/hkern.json]: true

File: svg/elements/hkern.json [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
```
Comment (teoli2003) [inline svg/elements/hkern.json]: true

File: svg/elements/hkern.json [hunk 6]
```diff
@@ -0,0 +1,289 @@
...
```
Comment (teoli2003) [inline svg/elements/hkern.json]: true

File: svg/elements/hkern.json [hunk 7]
```diff
@@ -0,0 +1,289 @@
...
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 8]
```diff
@@ -0,0 +1,289 @@
...
+            },
+            "firefox_android": {
+              "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 9]
```diff
@@ -0,0 +1,289 @@
...
+            },
+            "ie": {
+              "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 10]
```diff
@@ -0,0 +1,289 @@
...
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 11]
```diff
@@ -0,0 +1,289 @@
...
+            },
+            "edge_mobile": {
+              "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 12]
```diff
@@ -0,0 +1,289 @@
...
+                "version_added": null
+              },
+              "chrome": {
+                "version_added": null
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 13]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "edge_mobile": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 14]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "firefox": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 15]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "firefox_android": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 16]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "ie": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 17]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "chrome": {
+                "version_added": null
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 18]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "edge_mobile": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 19]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "firefox": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 20]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "firefox_android": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 21]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "ie": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 22]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "chrome": {
+                "version_added": null
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 23]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "edge_mobile": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 24]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "firefox": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 25]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "firefox_android": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 26]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "ie": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 27]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "chrome": {
+                "version_added": null
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 28]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "edge_mobile": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 29]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "firefox": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 30]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "firefox_android": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 31]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "ie": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 32]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "chrome": {
+                "version_added": null
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 33]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "edge_mobile": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 34]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "firefox": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 35]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "firefox_android": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

File: svg/elements/hkern.json [hunk 36]
```diff
@@ -0,0 +1,289 @@
...
+              },
+              "ie": {
+                "version_added": null
```
Comment (JeremiePat) [inline svg/elements/hkern.json]: Should be `false`

**Pull request merged**